express/blocks/columns/columns.css

```
@@ -120,4 +144,72 @@ main .columns .social.icon {
   main .columns.fullsize + h2 {
     margin-top: 0;
   }
-}
\ No newline at end of file
+}
+
+@media (max-width: 899px) {
+  main .columns-fullsize-center-container .columns.fullsize .hero-animation-overlay {
+    order: -1;
+  }
+
+  main .columns-fullsize-center-container .columns.fullsize > div > div {
+    display: grid;
+  }
+  
+  main .columns-fullsize-center-container .columns.fullsize > div > div .button-container {
+    grid-row: 2;
+  }
+
+  main .columns-fullsize-center-container .columns.fullsize .has-brand > div .button-container {
+    grid-row: 3;
+  }
+
+  main .columns-fullsize-center-container .columns.fullsize > div > div p {
+    margin: 16px 0;
+  }
+
+  main .columns-fullsize-center-container .columns.fullsize > div > div .button-container {
+    margin: 16px 0;
+  }
+
+  main .columns-fullsize-center-container .columns.fullsize > div > div .button-container a {
+    margin: 16px 0 0 0;
+  }
+
+  main .columns-fullsize-center-container .columns.fullsize .has-brand > div p:first-child {
+    margin: 0;
+  }
+
+  main .columns-fullsize-center-container .columns.fullsize > div > div .button-container.fixed-button {
+    margin: 0;
+  }
+
+  main .fixed-button.button-container {
+    display: block;
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    padding-bottom: 20px;
+    z-index: 2;
+    background: linear-gradient(180deg, transparent 0%, var(--color-white) 100%);
+    transform: translateY(101%);
+    transition: transform 1s;
+    pointer-events: none;
+  }
+
+  main .fixed-button.button-container.shown {
+    transform: none;
+    pointer-events: all;
+    bottom: 0;
+  }
+
+  body.has-fixed-button .button-container a {
```

This is hiding too many buttons on the page, see the empty "browse by category" (`link-list` block). I'd probably retrict this to just `a.primary`:
```suggestion
  body.has-fixed-button .button-container a.primary {
```
On a side note, I just noticed a problem with the `link-list` block: it fails to remove the `primary` class from its buttons, so that will have to be fixed as well to keep those buttons visible
https://github.com/adobe/spark-website/blob/main/express/blocks/link-list/link-list.js#L24